localization/sparse_mapping/scripts/copy_bag_topics.py

```diff
@@ -0,0 +1,13 @@
+import rosbag
```

Comment:
You might not know about rosbag filter, with that do we need this?

Reply:
oops didn't mean for that to be part of the commit. It was just more convenient for me than rosbag filter